### PR TITLE
starcos: added one more ATR of eGK

### PR DESCRIPTION
**Pull request by frankmorgner**
fixes https://github.com/OpenSC/OpenSC/issues/2460

##### Checklist

- [ ] Documentation is added or updated
- [ ] New files have a LGPL 2.1 license statement
- [ ] PKCS#11 module is tested
- [ ] Windows minidriver is tested
- [ ] macOS tokend is tested

### CODE REVIEW

File: src/libopensc/card-starcos.c [hunk 1]
```diff
@@ -44,6 +44,7 @@ static const struct sc_atr_table starcos_atrs[] = {
 	{ "3B:DF:96:FF:81:31:FE:45:80:5B:44:45:2E:42:4E:4F:54:4B:31:30:30:81:05:A0", NULL, NULL, SC_CARD_TYPE_STARCOS_V3_5, 0, NULL },
 	{ "3B:D9:96:FF:81:31:FE:45:80:31:B8:73:86:01:E0:81:05:22", NULL, NULL, SC_CARD_TYPE_STARCOS_V3_5, 0, NULL },
 	{ "3B:D0:97:FF:81:B1:FE:45:1F:07:2B", NULL, NULL, SC_CARD_TYPE_STARCOS_V3_4, 0, NULL },
+	{ "3B:D0:96:FF:81:B1:FE:45:1F:07:2A", NULL, NULL, SC_CARD_TYPE_STARCOS_V3_4, 0, NULL },
```
Comment (Jakuje) [inline src/libopensc/card-starcos.c:47]: Should we add this new ATR also to win32/customactions.cpp for minidriver?

Comment (Jakuje) [reply src/libopensc/card-starcos.c:47]: @frankmorgner ?

Comment (frankmorgner) [reply src/libopensc/card-starcos.c:47]: I don't think that's needed, because the cards don't currently have usable keys.

Comment (Jakuje) [reply src/libopensc/card-starcos.c:47]: Thanks for clarification.

**Pull request merged**